lib/engine/game/g_1848/game.rb

```diff
@@ -282,7 +282,7 @@ class Game < Game::Base
                 distance: 999,
                 price: 1100,
                 num: 20,
-                discount: { '4' => 300, '5' => 300, '6' => 300 },
+                discount: { '4' => 300, '4+' => 300, '5' => 300, '5+' => 300, '6+' => 300, '6' => 300 },
```

Comment:
suggestions on how to fix the method in engine/train.rb so i don't have to specify variants?
or maybe it makes sense to explicitly list them, as there might be games where the discount is different across variants of trains?

Reply:
I think it makes sense to explicitly list them